lib/Dialect/Torch/Transforms/GlobalizeObjectGraph.cpp

```diff
@@ -97,6 +101,32 @@ class ObjectGraphInfo {
   }
 
 private:
+  LogicalResult collectUsedSlots() {
```

Comment:
Can you please add .mlir test case for the case of an unused slot and also the case of a slot which is only set, but not get.

Reply:
Added the unused case in Basic.mlir and only set case in error.mlir